sdk/monitor/monitor-opentelemetry-exporter/samples/@azure/storage-blob/javascript/README.md

```diff
@@ -8,7 +8,7 @@ These sample programs show how to use the JavaScript client libraries for Azure
 
 ## Prerequisites
 
-The sample are compatible with Node.js >= 8.0.0, except for the samples that use the async `for await` syntax, which require Node.js >= 10.0.0.
+The sample are compatible with [LTS versions of Node.js](https://nodejs.org/about/releases/), except for the samples that use the async `for await` syntax, which require Node.js >= 10.0.0.
```

Comment:
```suggestion
The sample are compatible with [LTS versions of Node.js](https://nodejs.org/about/releases/)
```

Reply:
Nice catch! Fixed in https://github.com/Azure/azure-sdk-for-js/pull/16011/commits/7be616839e50b5db3eebfdf5220d00e47ee30802